### ISSUE TITLE
Overfitting when training my own data set

### ISSUE THREAD

**Issue by yonglianglan** [Rudrabha/Wav2Lip#303]
The training set loss drops, but the validation set rises

**Comment by xiao-keeplearning** [comment]
@yonglianglan Hi, can you suggest me how to build my own dataset, I am confused how to **keep the same crop face area setting as LRS2** ?

**Comment by yxqyyy** [comment]
hi, I have the same prolem, have you solved it?
